styles.css

```diff
@@ -137,16 +131,338 @@ p {
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.letter2 {
+  width: 327px;
+  height: 147px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.css {
+  width: 49px;
+  height: 40px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border: 1px solid #8993a4;
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+}
+
+.html {
+  width: 56px;
+  height: 40px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border: 1px solid #8993a4;
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+}
+
+.boostrap {
+  width: 95px;
+  height: 40px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border: 1px solid #8993a4;
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+}
+
+.ruby {
+  width: 61px;
+  height: 40px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border: 1px solid #8993a4;
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+}
+
+.seeproyect {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  justify-content: center;
+  flex-wrap: nowrap;
+  color: #fff;
+}
+
+.seeproyect2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 328px;
+  height: 48px;
+  background: #ff6b00;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  justify-content: center;
+  flex-wrap: nowrap;
+  color: #fff;
+}
+
+.grid {
+  display: grid;
+  justify-content: space-between;
+  align-items: center;
+  padding: 24px;
+  grid-template-rows: 24px 220px 20px 100px;
+}
+
+.aboutme {
+  width: 100%;
+  height: 1901px;
+  left: 1px;
+  top: 4137px;
+  background: #1c1a19;
+  flex-direction: column;
+  align-content: center;
+  display: flex;
+}
+
+.white {
+  color: #fff !important;
+  background: rgba(255, 255, 255, 0.24);
+}
```

Comment:
- [ ] Please avoid the use of `!important`. Create a professional cleaner code.
- [ ] Kindly use `background transparent` instead of `rgba`. That will remove the levels of black in your design.

Reply:
ok i erase it